outlines/models/openai.py

```diff
@@ -55,20 +67,20 @@ async def generate_base(
 
             return results
 
-        @functools.partial(outlines.vectorize, signature="(),(),(m),()->(s)")
+        @functools.partial(outlines.vectorize, signature="(),(),(m),(),()->(s)")
         async def generate_choice(
-            prompt: str, max_tokens: int, is_in: List[str], samples: int
+            prompt: str, max_tokens: int, is_in: List[str], samples: int, api_key: str
         ) -> Union[List[str], str]:
             """Generate a sequence that must be one of many options.
+            `
+                        .. warning::
```

Comment:
Why has this moved?

Reply:
I think formatter commit hook did this

Reply:
Hmm doesn't mess with the formatting now.